tutorials/scripting/resources.rst

```diff
@@ -229,7 +230,8 @@ Attach a script to it named ``bot_stats.gd`` (or just create a new script, and t
         using System;
         using Godot;
 
-        namespace ExampleProject {
+        namespace ExampleProject
+        {
             public class BotStats : Resource
```

Comment:
These classes are missing the `partial` modifier, while you are updating these examples it'd be nice to fix that too.

Reply:
Thanks, adding. Yeah, I wound up testing this in 3.x (for the Discord convo) and didn't think about this.